apps/golden-sample-app/src/app/app.component.html

```diff
@@ -83,7 +83,7 @@
           "
         >
           <bb-icon-ui
-            class="bb-layout__horizontal-nav-item-icon"
+            class="bb-layout__vertical-nav-item-icon"
```

Comment:
This is the only icon using a different class which causes a small misalignment after the RTL fix. I've switched it to follow other icons i.e. https://github.com/Backbase/golden-sample-app/blob/main/apps/golden-sample-app/src/app/app.component.html#L61.